pkg/kn/commands/service/export.go

```diff
@@ -34,17 +34,36 @@ import (
 	servingv1 "knative.dev/serving/pkg/apis/serving/v1"
 )
 
-var IGNORED_SERVICE_ANNOTATIONS = []string{
+// IgnoredServiceAnnotations defines the annotation keys which should be
+// removed from service annotations before export
```

Comment:
Is there a link on the API or docs repo where these are to be found?

Reply:
they are in the main branch of serving (and will be part of 0.21 release) https://github.com/knative/serving/pull/10539 
/cc @duglin might know better if the docs etc are in order